### PR TITLE
HTTPie 1.0.0

### PR DESCRIPTION
**Pull request by jkbrzt**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

Got an error unrelated to this formula:

```
 $ brew audit --strict httpie
==> Installing or updating 'rubocop' gem
Error: Unable to resolve dependency: user requested 'did_you_mean (= 1.0.0)'
```

----

Release:

https://github.com/jakubroztocil/httpie/releases/tag/1.0.0

### CODE REVIEW

File: Formula/httpie.rb [hunk 1]
```diff
@@ -3,36 +3,49 @@ class Httpie < Formula
...
   end
 
   resource "PySocks" do
-    url "https://files.pythonhosted.org/packages/fd/70/ba9982cedc9b3ed3c06934f1f46a609e0f23c7bfdf567c52a09f1296b8cb/PySocks-1.6.6.tar.gz"
-    sha256 "02419a225ff5dcfc3c9695ef8fc9b4d8cc99658e650c6d4718d4c8f451e63f41"
+    url "https://files.pythonhosted.org/packages/53/12/6bf1d764f128636cef7408e8156b7235b150ea31650d0260969215bb8e7d/PySocks-1.6.8.tar.gz"
+    sha256 "3fe52c55890a248676fd69dc9e3c4e811718b777834bcaab7a8125cf9deac672"
   end
 
+
```
Comment (fxcoudert) [inline Formula/httpie.rb]: One extra blank line.
And please restore the bottle block, it is handled automatically.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

**Comment by jakubroztocil** [comment]
@fxcoudert spaced removed, bottles returned

**Comment by Moisan** [comment]
```
        Error: 2 problems in 1 formula detected
httpie:
  * C: 8: col 3: `url` (line 8) should be put before `head` (line 6)
  * C: 9: col 3: `sha256` (line 9) should be put before `head` (line 6)
```

**Comment by jakubroztocil** [comment]
@Moisan updated 👍🏻

**Comment by Moisan** [comment]
@BrewTestBot test this please

**Review by fxcoudert — APPROVED** [review]

**Pull request closed without merging**